packages/discovery-react-components/src/components/DocumentPreview/components/PdfHighlight/PdfHighlight.tsx

```diff
@@ -123,7 +123,9 @@ const Highlight: FC<{
               className,
               shape.className,
               active && `${base}__item--active`,
-              active && activeClassName
+              active && activeClassName,
+              shape.facetId && `category_${shape.facetId} mf-highlight`,
```

Comment:
Regarding `mf-highlight`: This whole thing is a highlight. Do we need an additional class to specify that? Seems redundant.

Reply:
The benefit of the current form is that the specificity of the "matches found" style is higher than the default highlighting.

Reply:
But shouldn't this now be the "default" highlighting? I still feel like we're adding too many classes here unnecessarily.

Reply:
Currently color-coded highlighting is enabled if component `PdfViewerWithHighlight` is passed the optional `facetInfoMap`.

In the case of the Text view, it is enable if component `CIDocumentContent` is passed `combinedHighlights`.

Without the optional data, the original highlight is used.